doc/03-Configuration.md

```diff
@@ -35,4 +59,6 @@ database:
   password: icingadb
 redis:
   address: redis:6380
+logging:
+  level: info
```

Comment:
And config.yml.example?

Reply:
`config.yml` is subject to remove.

Reply:
* #375

Reply:
There is #377 now and I updated the `config.yml.example`.